content/en/dev-tut-dvm-deposit.md

```diff
@@ -41,7 +41,7 @@ add custom ternal configuration ws://t1.hkg.itering.com:9944/, save and reload a
 
 ## Recharge EVM address
```

Comment:
`Recharge EVM address` or `Deposit to DVM address` or `Deposit to a DVM address`?
@breeyin @wi1dcard